terminus/geometry/latlon.py

```diff
@@ -38,15 +38,22 @@ def __eq__(self, other):
         if self.lat != other.lat:
             return False
         else:
-            return (self.lon == other.lon) or (abs(self.lon) == 180 and abs(other.lon) == 180) or abs(self.lat) == 90
+            return (self.lon == other.lon) or \
+                   (abs(self.lon) == 180 and abs(other.lon) == 180) or \
+                abs(self.lat) == 90
+
+    def __repr__(self):
+        return '(lat, lon) = (%s, %s)' % (self.lat, self.lon)
 
     def sum(self, other):
         return LatLon(self.lat + other.lat, self.lon + other.lon)
 
     def translate(self, delta_lat_lon):
         """
-        Given LatLon point, returns a LatLon point corresponding to the place where an object would be after starting at the LatLon point
-        and moving first meters_long meters in the longitudinal direction, and then meterslat meters in the latitudinal direction.
+        Given LatLon point, returns a LatLon point corresponding to the place
+        where an object would be after starting at the LatLon point and moving
+        first meters_long meters in the longitudinal direction, and then
```

Comment:
Looks like the comment is outdated from the implementation (you no longer use meters_long / meterslat)